mmv1/third_party/terraform/acctest/vcr_utils.go

```diff
@@ -164,6 +173,31 @@ func VcrTest(t *testing.T, c resource.TestCase) {
 	c.Steps = steps
 
 	resource.Test(t, c)
+
+}
+
+func createFilesForReleaseDiffTest() (*os.File, *os.File, *os.File) {
+	// creates temporary file for the individual test, will be a temporary to store the output
+	var tempFile *os.File
+	var err error
+	tempFile, err = os.CreateTemp("", "release_diff_test_output_*.log")
+	if err != nil {
+		log.Fatalf("Error creating temporary file: %v", err)
```

Comment:
It might make more sense to throw a regular error here, as part of the return value. Same for the other files as well I would assume, since it seems like the diff test results can't effectively be consumed without them.